proto/grp.proto

```diff
@@ -61,6 +64,41 @@ message GetGroupResponse {
   string owned_domain = 3;
 }
 
+message GetGroupUsersRequest {
+  // Request context.
+  // This contains the group ID to get the users for.
+  context.RequestContext request_context = 1;
+
+  // Membership status filter (optional).
+  GroupMembershipStatus membership_status = 2;
+}
+
+message GetGroupUsersResponse {
+  // Users related to the group that match the request criteria.
+  repeated user_id.DisplayUser users = 1;
```

Comment:
rather than returning DisplayUser directly, could we return a proto containing DisplayUser and GroupMembershipStatus for each user?

Reply:
called `GroupUser`

Reply:
Done. Agree that it makes sense to add this now, since it'll be needed for full on user management later and we don't want to have to deprecate a field, probably.